torch/testing/_internal/common_utils.py

```diff
@@ -5566,3 +5566,27 @@ def load_inline(*args, **kwargs):
         return func(*args, load_inline=load_inline, **kwargs)
 
     return wrapper
+
+GPU_TYPES = ["cuda", "xpu"]
+
+# defines here before import torch._dynamo is for avoiding circular import
+# when get_gpu_type is imported from dynamo
```

Comment:
It seems like the `get_gpu_type` is only used in this file. If so, the function can be renamed as a private function.

Reply:
updated